WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -100,17 +102,6 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
 			}
 		}
 
-		private async Task WaitFor(Phase expectedPhase, CancellationToken cancellationToken)
-		{
-			// ideally this should await for a CompletionTask<RoundState> instead of
-			// iterate in this absurd way.
-			while (RoundState.Phase < expectedPhase)
-			{
-				await RefreshRoundAsync(cancellationToken).ConfigureAwait(false);
-				await Task.Delay(500).ConfigureAwait(false);
-			}
-		}
-
 		private async Task RefreshRoundAsync(CancellationToken cancellationToken)
 		{
 			// this code is part of a `RoundUpdater` background service that fetches this information
```

Comment:
This refresh method should be removed too because the updater has to have this info already,.

Reply:
ACK